src/pages/marketplaces/item.tsx

```diff
@@ -447,22 +451,71 @@ export const MarketplaceItem = () => {
 	const [selectedReply, setSelectedReply] = useState<Reply | undefined>()
 
 	const selectedReplyItemClean = replies.find((r) => r.from === provider)
+	const seekerProfile = useProfile(item?.owner)
+	const seekerPicture = useProfilePicture(
+		seekerProfile.profile?.pictureHash
+			? bufferToHex(seekerProfile.profile.pictureHash)
+			: ''
+	)
+	const seekerAvatar = useMemo(() => {
+		if (seekerPicture.picture) {
+			const blob = new Blob([seekerPicture.picture.data], {
+				type: seekerPicture.picture?.type,
+			})
+			return URL.createObjectURL(blob)
+		}
+	}, [seekerPicture.picture])
```

Comment:
There's a hook for this: `useProfilePictureURL`. I guess that also applies to the 3 other occurrences of this snippet. At least I don't think there are any differences?